app/views/application/_browsing_index.html.erb

```diff
@@ -14,4 +14,7 @@
     <hr>
   <% end %>
 <% end %>
+  <div class="pagination">
+    <%= will_paginate @obituaries, renderer: BootstrapPagination::Rails %>
```

Comment:
This is what produces the actual links to change pages.

Reply:
Wow, these gems really make things seem so simple!

Reply:
Nice Work!